tests/reader_writer_100hz_extended.test

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0"?>
+<launch>
+  <!-- Start Kvaser Can Bridge with Read/Write Ticks Enabled -->
+  <node name="kvaser_can_bridge" pkg="kvaser_interface" type="kvaser_can_bridge" ns="reader">
+    <param name="can_hardware_id" value="1" />
+    <param name="can_circuit_id" value="0" />
+    <param name="can_bit_rate" value="500000" />
+    <param name="enable_debug_ticks" value="true" />
+  </node>
+
+  <!-- Start Kvaser Can Bridge with Read/Write Ticks Enabled -->
+  <node name="kvaser_can_bridge" pkg="kvaser_interface" type="kvaser_can_bridge" ns="writer">
+    <param name="can_hardware_id" value="1" />
+    <param name="can_circuit_id" value="1" />
+    <param name="can_bit_rate" value="500000" />
+    <param name="enable_debug_ticks" value="true" />
+  </node>
+
+  <node name="topic_pub" type="rostopic" pkg="rostopic"
+    args="pub -r 100 /writer/can_rx can_msgs/Frame '{header: {seq: 0, stamp: {secs: 0, nsecs: 0}, frame_id: ''}, id: 2457, is_rtr: false, is_extended: true, is_error: false, dlc: 8, data: [0,0,0,0,0,0,0,0]}'" />
```

Comment:
I'd like to see this test with an ID that actually uses the full 29 bits allowed for the identifier, but it might not matter so long as it's a number > 2**11

Reply:
Same as above. Picked this one because it is 0x999 - just a random number. Again, might be better to use 0x15555555 / 357913941 / 0b10101010101010101010101010101.

Reply:
May want to do something similar with the payload.

Reply:
Should probably also verify that DLCs < 8 are respected too.

Reply:
IN FACT, we should probably just write a testing requirements list.